api/ssh.go

```diff
@@ -424,17 +425,17 @@ func (h *caHandler) SSHConfig(w http.ResponseWriter, r *http.Request) {
 func (h *caHandler) SSHCheckHost(w http.ResponseWriter, r *http.Request) {
 	var body SSHCheckPrincipalRequest
 	if err := ReadJSON(r.Body, &body); err != nil {
-		WriteError(w, BadRequest(errors.Wrap(err, "error reading request body")))
+		WriteError(w, errs.Wrap(http.StatusBadRequest, err, "error reading request body"))
```

Comment:
Looks ok, but most of the similar errors are written like:
```go
WriteError(w, errs.BadRequest(errors.Wrap(err, "error reading request body")))
```

Reply:
So, the trouble here is that we probably want to use our Wrap method which has a func type of `func(status int, err error, ....)` so we'd be writing `BadRequest(errs.Wrap(http.StatusBadRequest, err, ...`), repeating the bad request part.